user/views.py

```diff
@@ -43,12 +43,35 @@ def post(self, request):
             if User.objects.filter(account = data['account']).exists():
                 user = User.objects.get(account = data['account'])
                 if bcrypt.checkpw(data['password'].encode('utf-8'), user.password.encode('utf-8')):
-                    access_token = jwt.encode({'ID' : user.id}, SECRET['secret'], ALGORITHM['algorithm']).decode('utf-8')
+                    access_token = jwt.encode({'ID' : user.id}, SECRET_KEY['secret'], ALGORITHM['algorithm']).decode('utf-8')
```

Comment:
ID 보다는 USER_ID 로 정확하게 표현해주는게 좋을 것 같습니다~
```suggestion
                    access_token = jwt.encode({'USER_ID' : user.id}, SECRET_KEY['secret'], ALGORITHM['algorithm']).decode('utf-8')
```